server/sonar-server/src/main/java/org/sonar/server/issue/InternalRubyIssueService.java

```diff
@@ -154,60 +148,15 @@ public IssueQuery emptyIssueQuery() {
     return issueQueryService.createFromMap(Maps.<String, Object>newHashMap());
   }
 
-  @CheckForNull
-  public IssueFilterDto findIssueFilterById(Long id) {
-    return issueFilterService.findById(id);
-  }
-
-  /**
-   * Return the issue filter if the user has the right to see it
-   * Never return null
-   */
-  public IssueFilterDto findIssueFilter(Long id) {
-    return issueFilterService.find(id, userSession);
-  }
-
-  public boolean isUserAuthorized(IssueFilterDto issueFilter) {
-    try {
-      String user = issueFilterService.getLoggedLogin(userSession);
-      issueFilterService.verifyCurrentUserCanReadFilter(issueFilter, user);
-      return true;
-    } catch (Exception e) {
-      return false;
-    }
-  }
-
-  public boolean canUserShareIssueFilter() {
-    return issueFilterService.canShareFilter(userSession);
-  }
-
-  public String serializeFilterQuery(Map<String, Object> filterQuery) {
-    return issueFilterService.serializeFilterQuery(filterQuery);
-  }
-
-  public Map<String, Object> deserializeFilterQuery(IssueFilterDto issueFilter) {
-    return issueFilterService.deserializeIssueFilterQuery(issueFilter);
-  }
-
   public Map<String, Object> sanitizeFilterQuery(Map<String, Object> filterQuery) {
     return Maps.filterEntries(filterQuery, MatchIssueFilterParameters.INSTANCE);
   }
 
   /**
    * Execute issue filter from parameters
    */
-  public IssueFilterService.IssueFilterResult execute(Map<String, Object> props) {
-    return issueFilterService.execute(issueQueryService.createFromMap(props), toSearchOptions(props));
-  }
-
-  /**
-   * Execute issue filter from existing filter with optional overridable parameters
-   */
-  public IssueFilterService.IssueFilterResult execute(Long issueFilterId, Map<String, Object> overrideProps) {
-    IssueFilterDto issueFilter = issueFilterService.find(issueFilterId, userSession);
-    Map<String, Object> props = issueFilterService.deserializeIssueFilterQuery(issueFilter);
-    overrideProps(props, overrideProps);
-    return execute(props);
+  public List<IssueDoc> execute(Map<String, Object> props) {
+    return issueIndex.search(issueQueryService.createFromMap(props), toSearchOptions(props)).getDocs();
   }
 
   private static void overrideProps(Map<String, Object> props, Map<String, Object> overrideProps) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "overrideProps" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)